cfg/validate.go

```diff
@@ -177,6 +177,10 @@ func isValidMetricsConfig(m *MetricsConfig) error {
 	if m.StackdriverExportInterval != 0 && m.CloudMetricsExportIntervalSecs != 0 {
 		return fmt.Errorf("exactly one of stackdriver-export-interval and cloud-metrics-export-interval-secs must be specified")
 	}
+	const maxPortNumber = 65535
```

Comment:
nit: Does it make sense to declare it as a variable and not literal if it's used right here and only here ?

Reply:
This helps document the meaning of the number. Otherwise it'd look like a magic number.